web_teleop/package.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>web_teleop</name>
+  <version>0.0.0</version>
+  <description>The web_teleop package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="wxyelaine@todo.todo">wxyelaine</maintainer>
```

Comment:
Do either set a complete name and email, or copy the standard team lines that we're using in other packages.